Source/Core/Common/Timer.cpp

```diff
@@ -261,4 +261,13 @@ double Timer::GetDoubleTime()
 	return TmpTime;
 }
 
+std::string Timer::GetDateTimeFormatted(double time)
+{
+	time_t seconds = (time_t)time + (38 * 365 * 24 * 60 * 60);
+	struct tm * gmTime = localtime(&seconds);
+	char tmp[32];
+	strftime(tmp, 32, "%x %X", gmTime);
```

Comment:
http://www.cplusplus.com/reference/ctime/strftime/ suggests that the size parameter includes the null terminator.